bin/prettier.js

```diff
@@ -193,7 +191,11 @@ if (stdin) {
       if (write) {
         // Don't write the file if it won't change in order not to invalidate
         // mtime based caches.
-        if (output !== input) {
+        if (output === input) {
+          console.log("[ignore] %s", filename);
```

Comment:
Colors + duration for not updated files:

<img width="280" alt="screen shot 2017-02-04 at 00 04 44" src="https://cloud.githubusercontent.com/assets/4734297/22606505/92adb0e2-ea6d-11e6-9f16-cac723593fd2.png">

White background

<img width="280" alt="screen shot 2017-02-04 at 00 06 25" src="https://cloud.githubusercontent.com/assets/4734297/22606602/dd699af6-ea6d-11e6-8dbe-82deb04a8484.png">